code/gradle/release.gradle

```diff
@@ -117,64 +118,144 @@ ext {
 }
 
 task layoutNsisBase(type: Copy, dependsOn: copyToRoot) {
-	description="Assemble the required files for all PCGen installations"
-	doFirst {
-	  // Release notes are required!
-	  assert file(releaseNotes).exists()
-	} 
+    description="Assemble the required files for all PCGen installations"
+    doFirst {
+        // Release notes are required!
+        assert file(releaseNotes).exists()
+    } 
 
-	into nsisBaseFolder
-	from ("${projectDir}/installers/win-installer/Local") {
-		into "../Local"
-	}
-	from ("${projectDir}/installers/win-installer/") {
-		into "/docs/acknowledgments/"
-		include "PCGenLicense.txt"
-	}
-	from ("${projectDir}") {
-		include '*.jar'
-		include 'pcgen.exe'
-		include 'logging.properties'
-		include 'system/**' // Later we want to split out the game modes to optional
-		include 'docs/**'
-		include 'preview/**'
-		include 'characters/*.pcg'		
-	}
+    into nsisBaseFolder
+    from ("${projectDir}/installers/win-installer/Local") {
+        into "../Local"
+    }
+    from ("${projectDir}/installers/win-installer/") {
+        into "/docs/acknowledgments/"
+        include "PCGenLicense.txt"
+    }
+    from ("${projectDir}") {
+        include '*.jar'
+        include 'pcgen.exe'
+        include 'logging.properties'
+        include 'system/**' // Later we want to split out the game modes to optional
+        include 'docs/**'
+        include 'preview/**'
+        include 'characters/*.pcg'		
+    }
 
-	with baseLibs
-	with basePlugins
-	with nonPdfOutput
-	with baseData
+    with baseLibs
+    with basePlugins
+    with nonPdfOutput
+    with baseData
+}
+
+//See: https://github.com/TheBoegl/gradle-launch4j\
+launch4j {
+    mainClassName = 'pcgen.system.Main'
+    icon = "${projectDir}/installers/win-installer/Local/pcgen.ico"
+    outfile = "pcgen.exe"
+    jreMinVersion = '1.8.0'
+    jreMaxVersion = '1.8.9'
```

Comment:
I changed it due to the moving to JavaFX which needs Java 8+. That's where this change branched from. I can change it back and you guys update it when the rest is ready.